src/info/Info.ts

```diff
@@ -0,0 +1,135 @@
+export class DIVEInfo {
+    private static _supportsWebXR: boolean | null = null;
+
+    public static GetBrowser(): string {
```

Comment:
Are we sure we prefer to get the browser, browser version and system on our own? this checks are tricky and have to be maintained regularly. Adding a dependency library for this purpuse might be an option? they might be quite small, MIT and without subdependencies.

Reply:
I have thought about that as well. Found libs that are around 430kB in package size due to endless lines of RegEx expressions. We won't add those to DIVE. If we want to support such libs, we have to do it somewhere else in SW.

Reply:
Are u sure? I guess larger libs are those including more info than just browser version and os, but a minimal one should not be that large, maybe this one? ( ~4.8kB gzipped) https://github.com/bowser-js/bowser ?

Reply:
I guess our parsing requirements are quite low, so i think this should be fine :)

Reply:
![image](https://github.com/user-attachments/assets/c93bfaf5-d35f-4a2c-a10d-a9709e465c7e)
@dfrancos-hub seems to be like 217kB big on disk installed. 

I have to admit I don't really know how big it will be for the end user in a production version and built. But I am pretty sure it will not be like 5kB.

I like their `satisfies()` method:
![image](https://github.com/user-attachments/assets/2e299bbb-0cf2-4354-b01a-ece01268a602)

@Deristes what do you think?

Reply:
Hi @Deristes @ffrank913 am also ok with the current version. The suggestion of using a external lib is because of the "trickness" of user agent detection and maintainability. Actually, ideally we use feature detection instead of OS or browser sniffing, which is not recommended: https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent

Reply:
Yes, you are correct. I have checked the code so far.

The **WebXR** support is already checked via feature detection.

**ARQuickLook** is not checked via feature detection. I will change that. But let's keep the fallback check via userAgent. Just to be sure in the future.